### PR TITLE
Clarify description of private registry RKE option

### PR DESCRIPTION
**Pull request by catherineluse**
This PR addresses this issue:
https://github.com/rancher/docs/issues/1748

### CODE REVIEW

File: content/rancher/v2.x/en/cluster-provisioning/rke-clusters/options/_index.md [hunk 1]
```diff
@@ -58,7 +58,11 @@ If you want to see all the configuration options for a cluster, please click **S
 
 _Available as of v2.2.0_
 
-If you are using a private registry with authentication for your Docker images, please configure the registry in this section to allow the nodes to pull images from this registry. See [Private Registries]({{< baseurl >}}/rke/latest/en/config-options/private-registries/) for more information.
+The registry configuration here is applied during the provisioning of the cluster. This option tells Rancher to pull the [system]({{<baseurl>}}/rke/latest/en/config-options/system-images/) or [addon]({{<baseurl>}}/rke/latest/en/config-options/add-ons/) images, which are components needed to maintain the Kubernetes cluster.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/cluster-provisioning/rke-clusters/options/_index.md]: only system images are components needed for the k8s clusters. 

addons are just user requested workloads.

File: content/rancher/v2.x/en/cluster-provisioning/rke-clusters/options/_index.md [hunk 2]
```diff
@@ -58,7 +58,11 @@ If you want to see all the configuration options for a cluster, please click **S
...
+
+To deploy workloads that pull images from a private registry, you will need to [set up your own Kubernetes registry.]({{<baseurl>}}rancher/v2.x/en/k8s-in-rancher/registries/)
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/cluster-provisioning/rke-clusters/options/_index.md]: `you will need to set up your own Kubernetes registry for your project.`

File: content/rancher/v2.x/en/cluster-provisioning/rke-clusters/options/_index.md (hunk 1 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/cluster-provisioning/rke-clusters/options/_index.md]: updated

File: content/rancher/v2.x/en/cluster-provisioning/rke-clusters/options/_index.md (hunk 2 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/cluster-provisioning/rke-clusters/options/_index.md]: updated

**Pull request merged**